syntax = "proto3";

package pr;

message Metadata {
  int32 page_cnt = 1;
  int32 chunk_size = 2;
  repeated int32 out_link_cnts = 3;
}
